source/fundamentals/indexes.txt

```diff
@@ -87,13 +99,13 @@ server documentation on :manual:`Indexing Strategies </applications/indexes>` an
 Index Types
 -----------
 
-MongoDB supports a number of different index types to support querying your data. The following sections describe the
+MongoDB supports several different index types to support querying your data. The following sections describe the
 most common index types and provide sample code for creating each index type. For a full list of index types, see
-:manual:`Indexes </indexes/>`.
+:manual:`Indexes </indexes/>` in the Server manual.
 
 .. tip::
 
-   The MongoDB Kotlin driver provides the `Indexes <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/Indexes.html>`__ class that
+   The {+driver-short+} provides the `Indexes <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/Indexes.html>`__ class that
```

Comment:
Q: Are there multiple Index classes, and the driver provides this one? Or is there just one, and this sentence describes what it does?

If the latter, it should be "class, which" instead of "that"